.github/workflows/extract-api.yaml

```diff
@@ -8,6 +8,9 @@ on:
     branches:
       - master
       - release/*
+    paths-ignore:
+      - "**.md"
+      - docs/**
 
```

Comment:
Looks good so far! Really excited to have this integrated. 

Can we also include the codeowners file?

Reply:
I've added it to the excluded files. I tested it in my fork: https://github.com/mattbjordan/itwinjs-core-mergify/commit/bc2901aa2d76e8ea00dacab1bdb64ee72b6fd6d0. This commit shows that the status check skip is triggered and that the Azure Pipeline run is skipped after I made the same changes in the fork.

Reply:
I replaced one of the actions I was using and now its easier to add/remove files to the exclusion list. The codeowners files is still added.